src/pages/AdminDashboard/UserList/AdminToolbar/AdminToolbar.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton, Grid, Popper, Paper } from '@material-ui/core';
+import FilterListIcon from '@material-ui/icons/FilterList';
+
+import CheckBox from '../../../../components/CheckBox';
+import SearchToolbar from '../../../../components/SearchToolbar';
```

Comment:
make sure to use absolute imports here.  `import CheckBox from 'components/Checkbox'`